applications/StructuralMechanicsApplication/tests/test_perfect_plasticity_implementation_verification.py

```diff
@@ -235,7 +237,8 @@ def _create_reference_solution(current_model):
     }
     """)
 
-    out_parameters["output_file_name"].SetString("cl_test/test_perfect_plasticity_implementation_verification_reference.json")
+    input_filename = os.path.dirname(os.path.realpath(__file__))
+    out_parameters["output_file_name"].SetString(input_filename + "/cl_test/test_perfect_plasticity_implementation_verification_reference.json")
```

Comment:
could you please use the same as in the core for this?
https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/tests/test_gid_io.py#L11-L12

I think the current version won't work on Win

Reply:
Done

Reply:
But is basically the same with copy/paste